widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java

```diff
@@ -428,6 +428,9 @@ private void onMessageSent(VisitorMessage message) {
             Logger.d(TAG, "messageSent: " + message.toString() + ", id: " + message.getId());
             List<ChatItem> currentChatItems = new ArrayList<>(chatState.chatItems);
             changeDeliveredIndex(currentChatItems, message);
+            if (chatState.chatInputMode == ChatInputMode.SINGLE_CHOICE_CARD) {
```

Comment:
a comment here would help to understand, why this has to be done

Reply:
Since this is a public repository it is not a good practice to place ticket links into source code.
I believe a descriptive comment is enough.

Reply:
Jira link was missing from the description of the PR.

Reply:
updated